views/base.jade

```diff
@@ -60,7 +60,19 @@ html(ng-app="app")
             id: '#{ user.id }',
             href: '#{ contextPath }/#{ partyPath }/api/partyManagement/v2/individual/#{ user.id }',
             email: '#{ user.email }',
-            bearerToken: '#{ user.accessToken }'
+            bearerToken: '#{ user.accessToken }',
+            organizations: '#{ '{ "organizations":' + JSON.stringify(user.organizations) + '}' }',
+            currentUser: {
+              id: '#{ user.id }',
+              name: '#{ user.displayName }',
+              email: '#{ user.email }'
+            },
+            logInfo: {
```

Comment:
This object is redundant, you can directly use the information included in LOGGED_USER